src/components/Card/index.tsx

```diff
@@ -203,11 +204,16 @@ const TopLeftAdornment = styled.div`
   }
 `
 
+const Header = styled.div`
+  display: flex;
+  align-items: center;
+  gap: ${space[4]};
```

Comment:
I don't think we need this styling, I think this should become a normal `header` on line `352` without a styled component. You should apply this styling to the element that you are passing to the header prop 😄